Dokan/Dokan/Scenes/ProductDetails/ProductDetailsViewController.swift

```diff
@@ -10,27 +10,52 @@ import UIKit
 class ProductDetailsViewController: UIViewController {
     
     // MARK: Outlets
-
+    
+    @IBOutlet weak var buttonsView: ButtonsView!
+    
     // MARK: Properties
-        
+    
     private let viewModel: ProductDetailsViewModelType
-
+    private var addingToCart = false {
+        didSet {
+            if #available(iOS 15.0, *) {
+                buttonsView.addToCartButton.setNeedsUpdateConfiguration()
+            } else {
+                // Fallback on earlier versions
+            }
+        }
+    }
+    
+    private var addingToFavorite = false {
+        didSet {
+            if #available(iOS 15.0, *) {
+                buttonsView.addToFavoriteButton.setNeedsUpdateConfiguration()
+            } else {
+                // Fallback on earlier versions
+            }
+        }
+    }
+    
```

Comment:
instead of check version you can edit style to default and type to custom from button configuration
at xib file